site/lib/queries.ts

```diff
@@ -13,8 +13,7 @@ const hideFromProduction = IS_PRODUCTION
 
 const isDomainKlimadao = '(domain == "klimadao" || !defined(domain))';
 
-const localeFilter =
```

Comment:
The original requirements, is peculiar and not properly documented, sorry for that.
* English and Chinese are the only languages that we support for blog posts at the moment.
*  If the user is viewing the site in a supported language, only fetch localized blog posts.
*  If non-supported language, fetch English blog posts.

However as I'm writing this, I believe we should simplify this requirement.

* [ ] If the user is viewing the site in a supported language, fetch English AND localized posts.
* [ ] If the user is viewing the site in a non-supported language, fetch English blog posts.

I feel like chinese users will miss out on a lot of content that they would otherwise be interested in (and can read with Google Translate)